content/docs/coldchain/monitoring.md

```diff
@@ -175,22 +178,19 @@ Here's how the process looks (this is the mac desktop version):
 
 <div class="note">You may need to wait for the text file to be generated after connecting the Fridge-tag to your computer.</div>
 
-
-The following Berlinger devices are compatible with Open mSupply: 
-
-
-| Device              | Breach data displayed             |    Temperature graph plotted |
-| :------------------ | :------------------------------- | :------------------------------- |
-| **Q-tag CLm doc**       | Yes | Yes |
-| **Q-tag CLm doc D**            | Yes | Yes |
-| **Q-tag CLm doc Ice**        | Yes | Yes |
-| **Q-tag CLm doc Ice R**       | Yes   | Yes |
-| **Q-tag CLm doc L**    | Yes  | Yes |
-| **Q-tag CLm doc LR**     | Yes | Yes |
-| **Fridge-tag 2**     | Yes | No* |
-| **Fridge-tag 2E**     | Yes | No* |
-| **Fridge-tag 2L**     | Yes | Yes |
-| **Fridge-tag UL**     | Yes | Yes |
-
-*Logger device does not capture individual temperature logs
-
+The following Berlinger devices are compatible with Open mSupply:
```

Comment:
ooh, very nice!

Reply:
Auto format, didn't actually realise I'd done it...